extensions/guacamole-auth-ldap/src/main/java/org/apache/guacamole/auth/ldap/ConfigurationService.java

```diff
@@ -223,4 +223,36 @@ public int getMaxResults() throws GuacamoleException {
         );
     }
 
+    /**
+     * Returns whether or not LDAP aliases will be dereferenced,
+     * as configured with guacamole.properties.
+     * By default they will never be dereferenced.
+     *
+     * @return
+     *     An integer representing the status of of alias
```

Comment:
"An integer representing the status of alias dereferencing" is not clear in itself. The possible values of that integer, and the semantics of each, are not documented. Just documenting the values is insufficient, though. More on this below, but this would be a perfect spot to use an `enum`, that way all values are very strictly defined and documented.